aws-s3-transfer-manager/src/operation/download.rs

```diff
@@ -97,8 +97,8 @@ fn handle_discovery_chunk(
     permit: OwnedWorkPermit,
 ) -> u64 {
     let mut start_seq = 0;
-
     if let Some(stream) = initial_chunk {
+        let seq = handle.ctx.next_seq();
```

Comment:
`start_seq` and `seq` aren't connected anywhere here, should probably set `start_seq` to `seq` instead of hard coded to `1`

Reply:
Thanks, we would have to set start_seq = seq+1. I have added a current_seq() function which will return 0 or 1 depending upon we called next_seq or not.